bitbots_body_behavior/src/bitbots_body_behavior/main.dsd

```diff
@@ -2,68 +2,73 @@
 $AvoidBall
     NO --> $BallClose + distance:%behavior/body/ball_reaproach_dist
         YES --> $BallKickArea
-            NEAR --> @Stop + r:false, @LookAtBall + r:false, @StandAndWait + duration:1 + r:false, @LookForward + r:false, @KickBallDynamic
-            FAR --> @LookAtFieldFeatures, @GoToBall + target:gradient_goal
+            NEAR --> @ChangeAction + action:kicking, @Stop + r:false, @LookAtBall + r:false, @StandAndWait + duration:1 + r:false, @LookForward + r:false, @KickBallDynamic
+            FAR --> @ChangeAction + action:going_to_ball, @LookAtFieldFeatures, @GoToBall + target:gradient_goal
         NO --> @LookAtFieldFeatures, @GoToBall + target:gradient_goal + blocking:false + distance:%behavior/body/ball_far_approach_dist, @AvoidBallActive
```

Comment:
Here the action `@GoToBall` is also called, but without `@ChangeAction + action:going_to_ball`, so maybe we want to integrate this action switch into the @GoToBall` action which could include this.

Reply:
You can also call the going_to_ball action with a distance if you want to be a simple kind of supporter. Furthermore, I think it is clearer if all action changes are in the dsd file rather than spread between other actions and the dsd file

Reply:
While I agree that all things are at this same place it is also much more code in the dsd. Furthermore I think that the supporter position should be handled in a similar was to the defense position.

Reply:
This is not finished. It is only a draft and was implemented to test out simple team behavior as a proof of work